playbooks/roles/vault-ssh-signer/defaults/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+vault_policy_mgmt_token: testtoken
+vault_ssh_signer_ca: ca
+
+vault_config_dir: "/etc/vault"
+vault_tls_dir: "{{ vault_config_dir }}/tls"
+vault_tcp_tls_cert_file: "{{ vault_tls_dir }}/cert.pem"
+vault_tcp_tls_key_file: "{{ vault_tls_dir }}/key.pem"
+vault_tcp_tls_client_ca_file: "{{ vault_tls_dir }}/ca.pem"
+
+vault_env:
+    VAULT_TOKEN: "{{ vault_policy_mgmt_token }}"
```

Comment:
The policy management token can only has full capabilities on `sys/policies/*`. I think we'll need some "ssh-signer-manager" policy or something, whose token we could use here. Otherwise this role won't run properly.

Reply:
Oh, indeed. I incorrectly assumed it was a root token, my bad. I will change that to `vault_ssh_signer_mgmt_token` and test accordingly.

Reply:
done and this also  led to move shared variables in the vault group